src/devices/FrameBroadcaster/CMakeLists.txt

```diff
@@ -0,0 +1,41 @@
+# Copyright (C) 2006-2019 Istituto Italiano di Tecnologia (IIT)
+# All rights reserved.
+#
+# This software may be modified and distributed under the terms of the
+# BSD-3-Clause license. See the accompanying LICENSE file for details.
+
+yarp_prepare_plugin(FrameBroadcaster
+                    CATEGORY device
+                    TYPE yarp::dev::FrameBroadcaster
+                    INCLUDE FrameBroadcaster.h)
+
+if(ENABLE_FrameBroadcaster)
+
+  set(CMAKE_INCLUDE_CURRENT_DIR ON)
+
+  yarp_add_plugin(yarp_FrameBroadcaster FrameBroadcaster.cpp
+                                        FrameBroadcaster.h)
+
+   target_link_libraries(yarp_FrameBroadcaster PRIVATE YARP::YARP_OS
+                                                     YARP::YARP_sig
+                                                     YARP::YARP_dev
+                                                     YARP::YARP_math)
+  list(APPEND YARP_${YARP_PLUGIN_MASTER}_PRIVATE_DEPS YARP_OS
+                                                      YARP_sig
+                                                      YARP_dev
+                                                      YARP_math)
+
+  yarp_install(TARGETS yarp_FrameBroadcaster
+               EXPORT YARP_${YARP_PLUGIN_MASTER}
+               COMPONENT ${YARP_PLUGIN_MASTER}
+               LIBRARY DESTINATION ${YARP_DYNAMIC_PLUGINS_INSTALL_DIR}
+               ARCHIVE DESTINATION ${YARP_STATIC_PLUGINS_INSTALL_DIR})
+
+  yarp_install(FILES FrameBroadcaster.ini
+               COMPONENT ${YARP_PLUGIN_MASTER}
+               DESTINATION ${YARP_PLUGIN_MANIFESTS_INSTALL_DIR})
+
+  set(YARP_${YARP_PLUGIN_MASTER}_PRIVATE_DEPS ${YARP_${YARP_PLUGIN_MASTER}_PRIVATE_DEPS} PARENT_SCOPE)
+
+  set_property(TARGET yarp_FrameBroadcaster PROPERTY FOLDER "Plugins/Device/Fake")
```

Comment:
I don't think these are "fake" devices, therefore they should be in "Plugins/Device"